amenity-points.mss

```diff
@@ -2109,6 +2121,7 @@
   [feature = 'shop_car_repair'],
   [feature = 'shop_car_parts'],
   [feature = 'shop_cosmetics'],
+  [feature = 'shop_dairy'],
```

Comment:
the diff is not clean as dairy is not merged yet

Reply:
it is now :P sorry had issues with git not rebasing at proper location